lib/opponent.rb

```diff
@@ -0,0 +1,7 @@
+class Opponent
+
+  def computer_hand
+    ["Rock", "Paper", "Scissors"].sample
+  end
+
+end
```

Comment:
Like the use of another class for opponent, something I should have implemented. Nice following of the S in SOLID.